pkg/new-ui/v1beta1/frontend/src/app/shared/params-list/add-modal/add-modal.component.scss

```diff
@@ -21,3 +21,8 @@
     margin: 0 12px;
   }
 }
+
+input[type='number']::-webkit-outer-spin-button,
```

Comment:
Could we remove the use of `-webkit-outer-spin-button`?

It's non-standard https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-outer-spin-button, plus I also believe it's nice to include it. It makes it explicit that we are expecting a number.

Did you have a reason to remove the two arrows from the input, or was it just to ensure it will be the same as previously?

Reply:
I just meant to keep it same as previous.

Thank you for pointing out the standard!
I just removed the lines and pushed.

Reply:
Then we are good to roll!